### DISCUSSION TITLE
Classpath entry in Manifest Editor is modified, when Plugin is imported

### DISCUSSION THREAD

**Discussion by ceugster** [eclipse-pde/eclipse.pde#1366] [General]
Hi, up to now I am not 100% shure, when exactly the problem occurres: When I import a plugin with the import wizard (plugin and fragments, the imported plugin has the classpath entries in the runtime section of the manifest editor changed: Added the plugin symbolic name to the existing entries. If there has not already been a dot as entry, the plugin does not find anymore its files. 
![Bild 06 08 24 um 17 04](https://github.com/user-attachments/assets/59ee7f0c-9068-4864-9e3e-567eb4738096)

**Comment by laeubi** [comment]
@ceugster can you explain what exactly you are doing? Is there any example as a reproducer?

**Comment by ceugster** [comment]
@Laeubi 
Let me investigate to get more precise descriptions...
